### PR TITLE
Make Flyte scope all value configurable

### PR DESCRIPTION
**Pull request by milton0825**
Signed-off-by: Chao-Han Tsai <milton0825@gmail.com>

# TL;DR
Currently Flyteadmin requires `scope: all` to access the resources, this PR makes the scope name configurable. For example, we can make it `scope: flyte.all` or any other value

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [ ] Smoke tested
 - [X] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by milton0825** [comment]
@EngHabu  can you take a look
